app/models/concerns/has_file.rb

```diff
@@ -11,13 +11,29 @@ def file_mandatory?
     true
   end
 
+  def image_path(version = nil)
+    self.file_url(version).to_s.gsub(/^#{Regexp.escape(CONFIG['storage']['endpoint'])}/, CONFIG['storage']['public_endpoint'])
+  end
+
+  def file_path
+    self.image_path
+  end
+
+  def embed_path
+    self.image_path('embed')
+  end
+
   module ClassMethods
     def max_size
-      ENV['MAX_UPLOAD_SIZE'] ? Filesize.from("#{ENV['MAX_UPLOAD_SIZE']}B").to_f : (CONFIG['uploaded_file_max_size'] || 1.megabyte)
+      options = { var_name: 'MAX_UPLOAD_SIZE', config_name: 'uploaded_file_max_size', size: 1.megabyte}
+      if (self.name == 'UploadedVideo')
+        options = { var_name: 'MAX_VIDEO_SIZE', config_name: 'video_file_max_size', size: 20.megabyte }
+      end
+      ENV[options[:var_name]] ? Filesize.from("#{ENV[options[:var_name]]}B").to_f : (CONFIG[options[:config_name]] || options[:size])
```

Comment:
Plz follow the instructions given here to ignore this issue.

Reply:
Done